plugin/tcomment.vim

```diff
@@ -274,8 +274,8 @@ if g:tcomment_maps
         exec 'xmap '. g:tcomment_mapleader_comment_anyway .' <Plug>TComment_Comment'
     endif
     if g:tcomment_textobject_inlinecomment != ''
-        exec 'vmap' g:tcomment_textobject_inlinecomment ' <Plug>TComment_ic'
-        exec 'omap' g:tcomment_textobject_inlinecomment ' <Plug>TComment_ic'
+        exec 'vmap '. g:tcomment_textobject_inlinecomment .' <Plug>TComment_ic'
```

Comment:
There is no need for the extra period here.

Reply:
I did that to be consistent with the code a few lines above (all the other mappings use dots for string concat). But, I can remove it if you prefer (just let me know).